### ISSUE TITLE
(Partly) remove Grunt tasks

### ISSUE THREAD

**Issue by swissspidy** [wp-cli/scaffold-command#140]
A scaffolded plugin contains a bunch of Grunt tasks which I think are unnecessary and can be removed or replaced.

* `addtextdomain`  
Note sure if that one is really needed. There was quite a discussion about that feature in the i18n-command repo. If anything, this could be a `wp i18n addtextdomain` command. (See https://github.com/wp-cli/i18n-command/issues/25)
* `makepot`  
People can just use `wp i18n make-pot` to generate POT files.
* `wp_readme_to_markdown`  
I don't see the point in this. If you host your plugin on WordPress.org, you need to adhere to the very specific format for `readme.txt`. If you host or develop your plugin on GitHub, you usually have notes for developers, information about the build status, etc. IMHO it makes more sense to have two separate files — `readme.txt` and `readme.md` with different information.

Finally, I don't see why Grunt is needed for these simple tasks. Why can't these just be simple scripts in `package.json` (e.g. `"makepot": "wp i18n make-pot ."` would be one such script)? One dependency less to worry about.

**Comment by schlessera** [comment]
I agree, we should look into getting rid of Grunt altogether. The tool itself is already pretty dated, and just including it for such simple stuff is a bad practice.

**Comment by ernilambar** [comment]
I am trying to make PR for this. Would this be considered breaking to be included in the future `3.0.0` or we can include this now?

**Comment by danielbachhuber** [comment]
We can include it now.

**Comment by ernilambar** [comment]
PR - https://github.com/wp-cli/scaffold-command/pull/337
